### PR TITLE
[16.0][MIG] mail_composer_cc_bcc: Migrate to version 16.0

### PR DESCRIPTION
**Pull request by sonhd91**
Included port-forwarding from these PRs:
* https://github.com/OCA/social/pull/1198
* https://github.com/OCA/social/pull/1210
* https://github.com/OCA/social/pull/1223

### CODE REVIEW

**Review by TDu — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

File: mail_composer_cc_bcc/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Email CC and BCC",
+    "summary": "This module enables sending mail to CC and BCC partners in mail composer form.",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Social",
+    "website": "https://github.com/OCA/social",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
```
Comment (simahawk) [inline mail_composer_cc_bcc/__manifest__.py:10]:
```suggestion
    "author": "Camptocamp, Odoo Community Association (OCA)",
```
can you pls do this change in another commit that can be backported?

`mail_compose_cc_bcc: fix authorship`

Thanks!

File: mail_composer_cc_bcc/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "maintainers": ["hailangvn2023"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "preloadable": True,
+    "depends": [
+        "mail",
+        "account",
```
Comment (simahawk) [inline mail_composer_cc_bcc/__manifest__.py:18]: Q: why is this module depending on account?
I see it's only for overriding  a view for account invoice... can you please add to the ROADMAP that it should be moved out to a glue module?

File: mail_composer_cc_bcc/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "data": [
+        "views/res_company_views.xml",
+        "views/mail_mail_views.xml",
+        "views/mail_message_views.xml",
+        "views/mail_template_views.xml",
+        "wizards/account_invoice_send_views.xml",
+        "wizards/mail_compose_message_view.xml",
+    ],
+    "demo": [],
+    "qweb": [],
```
Comment (simahawk) [inline mail_composer_cc_bcc/__manifest__.py]:
```suggestion
```

File: mail_composer_cc_bcc/models/mail_mail.py [hunk 4]
```diff
@@ -0,0 +1,275 @@
+import ast
```
Comment (simahawk) [inline mail_composer_cc_bcc/models/mail_mail.py:4]: (c) missing

**Review by simahawk — CHANGES_REQUESTED** [review]

File: mail_composer_cc_bcc/__manifest__.py (hunk 2 above)
Comment (nilshamerlinck) [reply mail_composer_cc_bcc/__manifest__.py:18]: - this was to impact the `account.invoice.send` wizard as well in addition to the standard mail composer
- we took the assumption when working on v15 that most people have `account` installed in their db, but fair point, could be extracted to a more specific `mail_composer_cc_bcc_account`

File: mail_composer_cc_bcc/__manifest__.py (hunk 3 above)
Comment (sonhd91) [reply mail_composer_cc_bcc/__manifest__.py]: Branch updated, please check again

File: mail_composer_cc_bcc/__manifest__.py (hunk 1 above)
Comment (simahawk) [reply mail_composer_cc_bcc/__manifest__.py:10]: this is missing

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
pre-commit needs some love

**Comment by sonhd91** [comment]
> pre-commit needs some love

Branch updated, please check again
